packages/components/notification/src/composables/use-actions.ts

```diff
@@ -0,0 +1,93 @@
+import { type Ref, computed, ref } from 'vue'
+import { type MaybeComputedRef, resolveUnref as toValue } from '@vueuse/core'
+import { type Mutable, debugWarn, keysOf } from '@element-plus/utils'
+import type { ButtonProps } from '@element-plus/components/button'
+import type { NotificationAction, NotificationProps } from '../notification'
+
+type IntervalNotificationAction = {
+  label: string
+  button: Partial<ButtonProps>
+  disabled: Ref<boolean>
+  onClick: (event: MouseEvent) => Promise<void>
+}
+
+export function useActions(
+  actions: MaybeComputedRef<NotificationProps['actions']>,
+  closeNotification: () => void
+) {
+  const actions_ = computed(() => {
```

Comment:
```suggestion
  const _actions = computed(() => {
```
Internal variables may have a leading underscore.